docs/resources/service_account.md

```diff
@@ -0,0 +1,37 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_service_account Resource - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Services offer a way to provide programmatic, centrally-controlled, and consistent access controls. For more information, see Twingate's documentation https://www.twingate.com/docs/services.
```

Comment:
I know this is taken from the public docs, but let's do this everywhere: s/Services/Service Accounts/g

In this description, let's remove the sentence "For more information [...]"

Reply:
@alexmensch I've updated doc strings